Create CCC_TAC_Minutes-2022-04-21.md (#103)
confidential-computing/governance: TAC/Meetings/2022-04-21/CCC_TAC_Minutes-2022-04-21.md

## Links

* **Code of Conduct**: [code-of-conduct.confidentialcomputing.io](https://code-of-conduct.confidentialcomputing.io)
* **CCC Charter**: [charter.confidentialcomputing.io](https://charter.confidentialcomputing.io)
* **LF Training course on DEI**: [Inclusive Open Source Community Orientation (LFC102) (free)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
* **YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)

## Agenda and Minutes

### Welcome

Rep. Thaler opened the call at 7:05am PT. He welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation.

### Roll Call / Attendance

#### Voting members

> * Dave Thaler (Microsoft) P

Author: ```suggestion
* Dave Thaler (Microsoft)
```

> * Lily Sturman (Red Hat) P

Author: ```suggestion
* Lily Sturman (Red Hat)
```

* Simon Johnson (Intel) P
* Zhipeng (Howard) Huang (Huawei) P

> * Thomas Fossati (Arm) P

Author: ```suggestion
* Thomas Fossati (Arm)
* Dan Middleton (Intel)
```

> 

Reviewer: Please move `Dan Middleton (Intel)` from Attendees to Voting members

> #### Attendees

Reviewer 2: ```suggestion
#### Other Attendees
```

* Eric Voit (Cisco) P 

> * Penglin Yang (China Mobile) P

Author: ```suggestion
* Penglin Yang (China Mobile)
```

> * Dan Middleton (Intel) P

Author: ```suggestion
```

> * Steve Van Lare (Anjuna) P

Author: ```suggestion
* Steve Van Lare (Anjuna)
```

> * Jethro Beekman (Fortanix) P

Author: ```suggestion
* Jethro Beekman (Fortanix)
```

> * Henk Birkholz (Fraunhofer-Gesellschaft) P

Author: ```suggestion
* Henk Birkholz (Fraunhofer-Gesellschaft)
```

> * Vijay Nayani (Huawei) P

Author: ```suggestion
* Vijay Nayani (Huawei)
```

> * Xinxin Fan (IoTeX) P

Author: ```suggestion
* Xinxin Fan (IoTeX)
```

> * Sampo Sovio (Huawei) P

Author: ```suggestion
* Sampo Sovio (Huawei)
```

* Ravi Sahita (Rivos)

> 
> 
> #### Also Participating

Reviewer 2: ```suggestion
```

> * Ravi Sahita (Rivos) P

Author: ```suggestion
* Ravi Sahita (Rivos)
```

> * Kurt Taylor (Linux Foundation) P

Author: ```suggestion
* Kurt Taylor (Linux Foundation)
```

> * Shubhra Kar (Linux Foundation) P      

Author: ```suggestion
* Shubhra Kar (Linux Foundation)
```

### Meeting minute approvals

#### Approved by voting members

> **Resolved:** That the minutes of the [April 07, 2022](../2022-04-07/TAC_Minutes-2022-04-07.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.

Reviewer 2: ```suggestion
**Proposed:** That the minutes of the [April 07, 2022](../2022-04-07/TAC_Minutes-2022-04-07.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
```
I agree with Lily that the text above is confusing when it was not in fact approved, and I think it should be fixed in these minutes.  It would also be fine to remove this sentence and instead link to the minutes from line 82.

Author: This was inherited from Brian, I suspect that it is a rules of order thing, but I will be happy to reword it.

>  * Quorum was reached, however the minutes of April 7th meeting were not approved due to changes being needed.

Author: No the resolution was not passed (82), the changes are being worked and will be voted on in the next meeting.

>  * Quorum was reached, however the minutes of April 7th meeting were not approved due to changes being needed.

Author: I don't like the term "Resolved" either, maybe we can change that in future minutes to Motion to Approve, or Needs Vote, or something like that.

>  * Quorum was reached, however the minutes of April 7th meeting were not approved due to changes being needed.

Reviewer 3: I agree, thanks @krtaylor

#### Pending approval
 * None

### TAC Tech Talks

>  *  Code Scanning via LFX Security Shubhra Kar (Linux Foundation)

Reviewer 2: ```suggestion
 *  [Code Scanning via LFX Security](LFX%20Security.pdf) - Shubhra Kar (Linux Foundation)
```

 *  Dan - would like to hear feedback from Enarx
 *  DT - project mentors to work with projects to get code scan running on github
 *  Action item: KT to add all projects to LFX Security 
    *  https://security.lfx.linuxfoundation.org/

### Open Action Items

1. **[Stephen Walli, Dan Middleton, Aeva Black]** Develop a CoC escalation process, to be presented to TAC and governing board.
    * Dan Middleton has opened a PR for review https://github.com/confidential-computing/governance/pull/92

> 1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.

Reviewer 2: ```suggestion
1. **[Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
```

 * Jethro Beekman (JB) pointed out that this may be too expensive $150 for simple website, maybe go with another service not managed by LF and save money, can use the budget for another service
 * DM - make it easy for the project, up to $150 as a budget
 * DT - pointed out that the specific amount would be approved on a case-by-case basis
 * DT reviewed actual website
 * DT - would like to see a vote as Gramine is waiting for response
 * DM - asked for context from Eric Voit (EV) as their mentor, EV reported that they have discussed design and proposal
 * JB proposed website maintenence education (wordpress) by the LF for the same amount for upfront cost $500, for creation or education
 * Lily Sturmann (LS)  asked to vote upfront and monthly seperately, DT said it wouldn't help Gramine, because it will be a signup as a package deal
 * DT proposed up to $1800 year, $1000 standard policy for each project to use, vote on each project actual cost on a case-per-case basis as long as it falls into this budget
 * Simon Johnson (SJ) asked for a more specific policy with guidelines for what the TAC would be looking for and what would be approved and what would not

>  * DT said that he agreed with SJ and that consistancey across the projects is a good thing, SJ said that some clarification on overall professionsal website design would be good

Reviewer 2: ```suggestion
 * DT said that he agreed with SJ and that consistency across the projects is a good thing, SJ said that some clarification on overall professional website design would be good
```

 * DT called for a vote - slide 11 Proposal

>  **Approved** - Policy for new budget approved for website

Reviewer: Instead of referencing Slide 11 Proposal could you just put the text here that was approved?

 
 * DT Gramine is waiting, vote for approval for Gramine
 * DT called a vote for Gramine - no objections noted
 

>  **Approved** to have Gramine proceed with website request

Reviewer 2: Was there an AI to update the project benefits list?

### CCC Attestation SIG annual review

*  Thomas Fossati (TF) presenting

> *  DT and TF proposed vote for approval for Greg Kostal to take over Chair of Attestation from Larry, leads Microsoft Azure Attestation service - Vote ran by JB

Reviewer 2: ```suggestion
*  DT and TF proposed vote for approval for Greg Kostal to take over Chair of Attestation from Larry, leads Microsoft Azure Attestation service - Vote ran by JB as vice-chair, since DT recused from voting.
```

	**Approved** Greg Kostal attestation chair without objections, DT abstains

*  DM reminded everyone that there will be a Tuesday meeting for Attestation

### Outreach update

* No update from the outreach meeting

### Common terminology

>  * DT shared the termonology document

Reviewer 3: ```suggestion
 * DT shared the terminology document
```

 * EV reviewed the plan to take the current document and checkin to Github 
 * EV Trusted Codebase, is a official definition needed Trusted Computing Base (TCB)
 * DM noted that there may already be a definition for Trusted Computing Base

>  * EV commented on clairification on TCB definition questions in the document

Reviewer 2: ```suggestion
 * EV commented on clarification on TCB definition questions in the document
```

 * EV marketing communications - comment on confidential VM
 * EV where does TDX fit, where should it appear, need clarification from Intel, clarified by DM, please add TDX

>  * DT remindeed that TAC doesn't take a position on vendor products, maybe change the word "known" to "claimed", Ravi Sahita (RS) commented he could add RISC-V directions, EV to make changes

Reviewer 2: ```suggestion
 * DT reminded that TAC doesn't take a position on vendor products, maybe change the word "known" to "claimed", Ravi Sahita (RS) commented he could add RISC-V directions, EV to make changes
```

 * DT noted that the definitions will never be complete, agreed on RISC-V additions as long as it is an easy add, else we can add it in a future revision
 * DT agreed on expanding the definition for TCB
 * DT asked KT about timeline for LF creative services to produce the whitepaper, preferred initial format for conversion
 * EV, DT discussed where to push initial document for comments, agreed to push to CCC governance repo, maybe in a whitepaper or terminology directory
 * Steve Van Lare (SVL) comments on confidential shared responsibility relating to cloud infrastructure, making sure we have a common definition

>  * DT suggested a change terminology as it is not specific to cloud computing

Reviewer 2: ```suggestion
 * DT suggested a change in terminology as it is not specific to cloud computing
```
